app/Jobs/Service/Install/VerifyInstallation.php

```
@@ -0,0 +1,260 @@
+    /**
+     * Execute the job.
+     */
+    public function handle(): void
+    {
+        $domain = Str::slug($this->service->customer->entreprise). '.'.config('batistack.domain');
+        $domainPath = '/www/wwwroot/'.$domain;
+
+        // Configuration SSH
+        $sshHost = config('batistack.ssh.host');
+        $sshUser = config('batistack.ssh.user');
+        $sshKey = config('batistack.ssh.private_key_path');
```

PHPMD (2.15.0)

Avoid unused local variables such as '$sshKey'. (Unused Code Rules)

(UnusedLocalVariable)